guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -5,18 +5,9 @@ include::common/header.adoc[]
 
 = {UpgradingDocTitle}
 
-In this guide, the terms upgrade, update, and migrate have the following meanings:
-
-Upgrading::
-The process of advancing your {ProjectServer} and {SmartProxyServer} installations from a y-stream release to the next, for example {Project} {ProjectVersionPrevious} to {Project} {ProjectVersion}.
+The term upgrade refers to the process of advancing your {ProjectServer} and {SmartProxyServer} installations from a y-stream release to the next, for example {Project} {ProjectVersionPrevious} to {Project} {ProjectVersion}.
```

Comment:
FYI, this has been addressed here - https://github.com/theforeman/foreman-documentation/pull/2279

I would suggest you go through it once so that you can save time on some of the parts. 🙂

Reply:
ack @mjivraja 
reverted the changes :)